### PR TITLE
Get the PACTA frontend/backend ready for deployment

### PR DESCRIPTION
**Pull request by bcspragu**
Does things like:

- Create sops configs and use them
- Set the registry to push to
- Verify the JWT key ID
- Update frontend configs

### CODE REVIEW

File: cmd/server/main.go [hunk 1]
```diff
@@ -80,6 +78,14 @@ func run(args []string) error {
 		}
 	}
 
+	// Pub is the key we use to authenticate signatures on user auth tokens.
+	logger.Info("Loading sops secrets", zap.String("sops_path", *sopsPath))
+	sec, err := secrets.LoadPACTA(*sopsPath)
+	if err != nil {
+		return fmt.Errorf("failed to decrypt secrets: %w", err)
+	}
+	pub := sec.AuthVerificationKey.PublicKey
```
Comment (gbdubs) [inline cmd/server/main.go:87]: meganit: inline?

File: go.mod [hunk 2]
```diff
@@ -10,134 +10,127 @@ require (
...
-	github.com/mitchellh/mapstructure v1.4.3 // indirect
-	github.com/mitchellh/reflectwalk v1.0.2 // indirect
+	github.com/mitchellh/mapstructure v1.5.0 // indirect
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826 // indirect
-	github.com/oklog/run v1.1.0 // indirect
-	github.com/pierrec/lz4 v2.6.1+incompatible // indirect
+	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
+	github.com/rogpeppe/go-internal v1.11.0 // indirect
+	github.com/russross/blackfriday/v2 v2.1.0 // indirect
```
Comment (gbdubs) [inline go.mod:110]: sketchy name for a markdown editor but ok

**Review by gbdubs — APPROVED** [review]
Looks great. Thanks!

File: cmd/server/main.go (hunk 1 above)
Comment (bcspragu) [reply cmd/server/main.go:87]: GC, done

**Pull request merged**